app/Http/Controllers/PhotosController.php

```diff
@@ -186,7 +181,7 @@ public function store (Request $request)
         $latitude = $latlong[0];
         $longitude = $latlong[1];
 
-        $revGeoCode = $this->reverseGeocodeAction->run($latitude, $longitude);
+        $revGeoCode = app(ReverseGeocodeLocationAction::class)->run($latitude, $longitude);
```

Comment:
any reason why this changed?

Reply:
Yes, I added a fake for this action, which allows me more options to properly mock the geocoding action. I don't need to mock this class in every test anymore. Using dependency injection without an interface forces me to use app() in this case. Nothing wrong with it though, this code piece is perfectly fine.